tests/e2e/conftest.py

```diff
@@ -25,7 +25,7 @@ def init_env():
 
     # TODO: Remove this setting after lmnet.environment has been refactored.
     envs = {
-        "DATA_DIR": blueoil_dir,
+        "DATA_DIR": os.path.join(blueoil_dir, "lmnet", "tests", "fixtures", "datasets"),
```

Comment:
```suggestion
        "DATA_DIR": os.path.join(blueoil_dir, "lmnet/tests/fixtures/datasets"),
```

Reply:
I think we should do `os.path.join("a", "b", "c")` instead of `("a", "b/c")` for better cross-platform compatibility

Reply:
I see.